src/widgetastic_patternfly5/components/date_and_time/calendarmonth.py

```diff
@@ -0,0 +1,101 @@
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.keys import Keys
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import Widget
+from widgetastic.xpath import quote
+
+from ..menus.select import Select
+
+
+class DisabledDate(Exception):
+    pass
+
+
+class BaseCalendarMonth:
+    """Represents calendar month component.
+
+    https://www.patternfly.org/components/date-and-time/calendar-month
+    """
+
+    CALENDAR_HEADER = ".//div[@class='pf-v5-c-calendar-month__header']"
```

Comment:
Yesterday we decided to use compatible locators with PF4 and PF5:
`.//div[contains(@class, '-c-calendar-month__header')]`